src/vcpkg/vcpkgpaths.cpp

```diff
@@ -731,6 +731,10 @@ namespace vcpkg
             {
                 for (auto&& path : fs.get_regular_files_non_recursive(triplets_dir, VCPKG_LINE_INFO))
                 {
+                    if (path.filename() == ".DS_Store")
```

Comment:
I think it would be simpler to just require the extension to be `.cmake` (note: case insensitive!)